package.json

```diff
@@ -68,25 +68,7 @@
     "webpack": "3.8.1",
     "zxcvbn": "^4.2.0"
   },
-  "devDependencies": {
```

Comment:
let's save this part of the change for https://github.com/CenterForOpenScience/osf.io/pull/9661 to limit where our yarn.lock deltas show up?

Reply:
nevermind, we don't really care about individual change deltas